ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -27,48 +30,58 @@ public static class ChangeFeedProcessor
         /// <summary>
         /// Name of the Event Hub.
         /// </summary>
-        private static readonly string EventHubName = "event-hub1";
+        private const string EventHubName = "event-hub1";
 
+        // Create producer client to send change feed events to event hub.
+        private static readonly Lazy<EventHubProducerClient> lazyClient = new Lazy<EventHubProducerClient>(InitializeEventHubProducerClient);

```

Comment:
Is there a reason that we're not using [dependency injection](https://docs.microsoft.com/azure/azure-functions/functions-dotnet-dependency-injection) to configure the producer.   That would be the recommended approach.  In `ConfigureServices`, the `AddEventHubProducerClient` extension can be used to register the producer as a singleton and ensure proper clean-up.

Reply:
@jsquire - I have updated, please review.